README.md

```diff
@@ -1,51 +1,39 @@
 
 ## Enter LimeLight
 
-**LimeLight** is a disaster management ecosystem which allows rescue operations to be carried out in the best possible manner.
-It would comprise of an android application and a central server.
+**LimeLight** is a disaster management ecosystem which allows rescue operations to be carried out in the best possible manner. It would comprise of an android application and a central server.
 
-The first step is locating victims. The mobile phones of victims will continuously send POST requests to a central server with
-their location as the payload. Their location would be rendered in a Google Map Interface in the rescuer's phone. A cluster
-of locations would comprise a rescue zone which would be circular in nature and also rendered on the Map Interface. Now rescue
-operations can be carried out on a zonal basis. In case there is no cellular connectivity in the area, the victims would be located
-via their phone's **bluetooth**. The range of a bluetooth is approximately 100 metres. Whenever a rescue worker steps within
-100 metre radius of a victim, he would be notified of the victim's location.
+The first step is locating victims. The mobile phones of victims will continuously send POST requests to a central server with their location as the payload. This location polling can be implemented in two ways: it could either be a persistent sending of location data at regular intervals, or it could be triggered by a governing authority at a specific location in the event of a disaster.
 
-The second step is carrying out the rescue operations in the most optimal manner. It can done by designating specific roles to
-the rescue workers and alloting manpower to each rescue zone. There would be mainly 3 roles:-
+These locations would be rendered in a Google Map Interface in the rescuers' phones. A cluster of locations would comprise a rescue zone. By clicking on a marker in the map interface, information such as the number of devices in the cluster shall be visible. Now rescue operations can be carried out on a zonal basis. In case, there is no internet connectivity at a disaster site, the victims would be located via their phone's **bluetooth**. The range of a bluetooth is approximately 100 metres. Whenever a rescue worker steps within 100 metre radius of a victim, he would be notified of the victim's presence.
 
-* Scout - One who surveys the area and reports the damage and status of victims
-* Medic - One who is responsible for providing first-aid treatment to the victims
-* Lifter - One who removes wreckage and pulls out victims from underneath
+The second step is carrying out the rescue operations in the most optimal manner. There are mainly two kinds of people involved in the relief effort:
 
-There are only 3 roles in the initial draft but the infrastructure of the application would be such that it would allow defining any number of custom roles as per the requirements.
```

Comment:
The role system should be extensible. By hardcoding those roles we are restricting the ecosystem. Hence this line is necessary